### PR TITLE
Mappings

### PR DESCRIPTION
**Pull request by TehcJS**

### CODE REVIEW

**Comment by TehcJS** [comment]
y nobody review ;-;

File: mappings/net/minecraft/class_123.mapping [hunk 1]
```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_123 net/minecraft/world/biome/source/BiomeSource
```
Comment (BoogieMonster1O1) [inline mappings/net/minecraft/class_123.mapping:1]: nice nice nice 👍

**Review by BoogieMonster1O1 — APPROVED** [review]
@hYdos @shedaniel review this or i merge :yeefrickinhaw:

File: mappings/net/minecraft/entity/player/PlayerEntity.mapping [hunk 2]
```diff
@@ -77,6 +79,8 @@ CLASS net/minecraft/class_1963 net/minecraft/entity/player/PlayerEntity
 		ARG 3 setSpawn
 	METHOD method_8001 decrementXp (I)V
 		ARG 1 xp
+	METHOD method_8002 canToolBreak (Lnet/minecraft/class_160;)Z
```
Comment (shedaniel) [inline mappings/net/minecraft/entity/player/PlayerEntity.mapping:82]: isUsingEffectiveTool?

File: mappings/net/minecraft/entity/player/PlayerInventory.mapping [hunk 3]
```diff
@@ -9,10 +9,12 @@ CLASS net/minecraft/class_1962 net/minecraft/entity/player/PlayerInventory
 		ARG 1 player
 	METHOD method_7945 damageArmor (F)V
 		ARG 1 armor
+	METHOD method_7946 getMiningSpeed (Lnet/minecraft/class_160;)F
```
Comment (shedaniel) [inline mappings/net/minecraft/entity/player/PlayerInventory.mapping:12]: getBlockBreakingSpeed

**Review by shedaniel — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/entity/player/PlayerInventory.mapping (hunk 3 above)
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/entity/player/PlayerInventory.mapping:12]: no
we use `getMiningSpeedMultiplier` everywhere else
not `getBlockBreakingSpeedMultiplier` concen

Comment (shedaniel) [reply mappings/net/minecraft/entity/player/PlayerInventory.mapping:12]: That's official yarn

Comment (BoogieMonster1O1) [reply mappings/net/minecraft/entity/player/PlayerInventory.mapping:12]: And this isn't

Comment (BoogieMonster1O1) [reply mappings/net/minecraft/entity/player/PlayerInventory.mapping:12]: welcome to `Container`ville

Comment (shedaniel) [reply mappings/net/minecraft/entity/player/PlayerInventory.mapping:12]: Boy look at PlayerEntity#getBlockBreakingSpeed

Comment (BoogieMonster1O1) [reply mappings/net/minecraft/entity/player/PlayerInventory.mapping:12]: its
fine

Comment (BoogieMonster1O1) [reply mappings/net/minecraft/entity/player/PlayerInventory.mapping:12]: you wouldn't want refactoring container to handler, would you
even though handler is official yarn

Comment (TehcJS) [reply mappings/net/minecraft/entity/player/PlayerInventory.mapping:12]: no change :concern:

File: mappings/net/minecraft/entity/player/PlayerEntity.mapping (hunk 2 above)
Comment (TehcJS) [reply mappings/net/minecraft/entity/player/PlayerEntity.mapping:82]: too long :P

Comment (BoogieMonster1O1) [reply mappings/net/minecraft/entity/player/PlayerEntity.mapping:82]: `isUsingEffectiveTool` is better
use it

File: mappings/net/minecraft/entity/player/PlayerInventory.mapping (hunk 3 above)
Comment (TehcJS) [reply mappings/net/minecraft/entity/player/PlayerInventory.mapping:12]: no

**Review by hYdos — APPROVED** [review]

**Pull request merged**